client/sites/components/sites-dataviews/index.tsx

```diff
@@ -58,34 +57,6 @@ const DotcomSitesDataViews = ( {
 	const { __ } = useI18n();
 	const userId = useSelector( getCurrentUserId );
 
-	// Scroll to selected site in the list when in list view.
```

Comment:
Thanks.

The list layout already has a scroll into view behavior: it's just different. It scrolls the minimum amount so the item is visible.

I've prepared https://github.com/WordPress/gutenberg/pull/67300 to explore how it'd look like to scroll to top when first mounting the item (proxy for switching the layout), though I'm unconvinced this is an improvement. I want to hear feedback from folks.